.gitignore

```diff
@@ -17,3 +17,5 @@ tests/freetype/objs/*.lo
 third_party/lzma.js/lzip/*.o
 third_party/lzma.js/lzma-native
 third_party/lzma.js/lzma-native.exe
+
```

Comment:
It was from the SDK. I use ```emsdk``` to install and activate the SDK at first. And later based on the online doc (or what I have understood it), cd to "emscripten/incoming" directory and add another remote pointing to my own fork before starting to make my changes. I believe I notice "tools/optimizer_build/" is untracked at the very beginning.